express_relay/sdk/python/express_relay/svm/limo_client.py

```diff
@@ -198,7 +200,9 @@ async def take_order_ix(
             taker: The taker's public key
             order: The order to fulfill
             input_amount_decimals: The amount of input tokens to take multiplied by 10 ** input_mint_decimals
+            output_amount_decimals: The amount of output tokens to take multiplied by 10 ** output_mint_decimals
```

Comment:
I think both this comment and the one above is wrong, since we are multiplying by (10**mint_decimals) inside this function and the searcher should not do it outside

Reply:
yeah the phrasing here is ambiguous, let me fix that